netsim/extra/firewall.zonebased/defaults.yml

```diff
@@ -0,0 +1,28 @@
+# firewall.zonebased
+#
+---
+devices:
+  vsrx:
+    features.firewall.zonebased: True
+  vyos:
+    features.firewall.zonebased: True
+
+attributes:
+  node:
+    firewall.zonebased:
+      default_rules: { type: list, _subtype: _fw_rule_entry }
+  interface:
+    firewall.zone: { type: str }
```

Comment:
Is it OK to allow any string value or would it be better to insist on IDs (`type: id`)?